### ISSUE TITLE
Orca Pytorch Estimator can't fit RNN Models

### ISSUE THREAD

**Issue by jing-xu** [intel-analytics/analytics-zoo#3210]
Currently both the torch_distributed backend and bigdl backend could only support `output = model(data)` type of applications. For RNN models, we need `output, hidden = model(data, hidden)` in the training and evaluation process.

**Comment by hkvision** [comment]
@jason-dai @yangw1234 @qiuxin2012 Any ideas?

**Comment by leonardozcm** [comment]
tf2 also meets the same problem.

**Comment by qiuxin2012** [comment]
how about the loss? Do you need the hidden to compute loss?

**Comment by jing-xu** [comment]
> how about the loss? Do you need the hidden to compute loss?

@qiuxin2012 To my understanding, the hidden will not be directly used in compute loss, but it will contribute to the training and predicting process.

**Comment by hkvision** [comment]
https://github.com/pytorch/examples/blob/master/word_language_model/main.py
